### ISSUE TITLE
Stripe payments

### ISSUE THREAD

**Issue by samnabi** [samnabi/shopkit#70]
Add Stripe as another gateway option.
https://stripe.com/docs/checkout/tutorial
https://stripe.com/docs/charges

**Comment by Mojo90** [comment]
@samnabi 
I added stripe as a payment method with the appropriate keys/tokens.
In stripe I activated not only Credit Card but also Giropay, Sepa direct debit and SOFORT. Should that be shown automatically within the payment process when selecting stripe? Because for me I can only see the following (in test mode, did not try it yet in productive mode):

<img width="255" alt="screen shot 2017-12-02 at 14 47 16" src="https://user-images.githubusercontent.com/2122156/33516234-bfba58f2-d76f-11e7-810c-70e5612bda46.png">

So I can only pay with credit card and not the other mentioned payment methods of stripe.
